docs/multicluster/antctl.md

```diff
@@ -1,11 +1,11 @@
 # Antctl Multi-cluster commands
 
 Starting from version 1.6.0, Antrea supports the `antctl mc` commands, which can
-collect information from a leader cluster in a ClusterSet, for troubleshooting
-issues in an Antrea Multi-cluster ClusterSet.
+collect information from a leader cluster in a ClusterSet for troubleshooting
+issues in an Antrea Multi-cluster ClusterSet, create and delete resources in an 
+Antrea Multi-cluster ClusterSet, and so on.
```

Comment:
Maybe highlight all commands can run from out-of-cluster only.

Reply:
Add a few description like this about how to switch different clusters context? https://github.com/antrea-io/antrea/blame/main/docs/antctl.md#L81